Description: Change to 0.9.13-1 quilt format
 cutting out debian directory into separate debian branch seems
 to get much more difficult than expected ...
 The patch beeing directed backwards is big surprise, i.e.
 new entries are mareked with '-', old ones with '+'
Last-Update: 2015-02-10
Author: Johannes Hubertz <johannes@hubertz.de>

--- iptables-optimizer-0.9.13.orig/README.txt
+++ iptables-optimizer-0.9.13/README.txt
@@ -6,9 +6,9 @@ Optimize kernel's iptables ruleset by us
 
 Author:     Johannes Hubertz <johannes@hubertz.de>
 
-Date:       2015-02-10
+Date:       2015-01-16
 
-Version:    0.9.13
+Version:    0.9.12
 
 License:    GNU General Public License version 3 or later
 
--- iptables-optimizer-0.9.13.orig/docs/source/conf.py
+++ iptables-optimizer-0.9.13/docs/source/conf.py
@@ -42,7 +42,7 @@ master_doc = 'index'
 
 # General information about the project.
 project = u'iptables-optimizer'
-copyright = u'2014, 2015, Johannes Hubertz'
+copyright = u'2014, Johannes Hubertz'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
@@ -51,7 +51,7 @@ copyright = u'2014, 2015, Johannes Huber
 # The short X.Y version.
 version = '0.9'
 # The full version, including alpha/beta/rc tags.
-release = '0.9.12'
+release = '0.9.13'
 
 # The language for content autogenerated by Sphinx. Refer to documentation
 # for a list of supported languages.
--- iptables-optimizer-0.9.13.orig/iptables-optimizer
+++ iptables-optimizer-0.9.13/iptables-optimizer
@@ -3,7 +3,7 @@
 # License:  GNU General Public License version 3 or newer
 # Author:   Johannes Hubertz <johannes@hubertz.de>
 # Version:  0.9.13
-# Date:     2015-02-10
+# Date:     2015-01-16
 #
 # Fullname: /usr/sbin/iptables-optimizer
 #           /usr/sbin/ip6tables-optimizer  (works a symlink)
@@ -18,9 +18,9 @@
 #
 #    -h    hint for options
 #
-#    -v    turns on logging, twice is more verbose
+#    -i    verbose logging reduced to INPUT and OUTPUT chains
 #
-#    -w    verbose logging reduced to INPUT and OUTPUT chains
+#    -v    turns on logging, twice is more verbose
 #
 ##############################################################
 # iptables-optimizer runs in discrete steps:
--- iptables-optimizer-0.9.13.orig/iptables-optimizer-functions
+++ iptables-optimizer-0.9.13/iptables-optimizer-functions
@@ -3,7 +3,7 @@
 # License:  GNU General Public License version 3 or newer
 # Author:   Johannes Hubertz <johannes@hubertz.de>
 # Version:  0.9.13
-# Date:     2015-02-10
+# Date:     2015-01-16
 #
 # these functions are sourced in iptables-optimizer and it's tests
 
--- iptables-optimizer-0.9.13.orig/iptables_optimizer.py
+++ iptables-optimizer-0.9.13/iptables_optimizer.py
@@ -8,7 +8,7 @@
     in relation to usage (packet counters)
 
 Author:     Johannes Hubertz johannes@hubertz.de
-Date:       2015-02-10
+Date:       2015-01-16
 Version:    0.9.13
 License:    GNU General Public License version 3 or later
 
--- iptables-optimizer-0.9.13.orig/man/ip6tables-optimizer.8
+++ iptables-optimizer-0.9.13/man/ip6tables-optimizer.8
@@ -1,5 +1,5 @@
 
-.TH ip6tables-optimizer 8 "February 2015" "Johannes Hubertz" " "
+.TH ip6tables-optimizer 8 "September 2014" "Johannes Hubertz" " "
 
 .SH NAME
 ip6tables-optimizer \- optimize ip6tables filter-chains in kernel depending on their usage counters
@@ -114,6 +114,6 @@ GNU General Public License version 3 or
 ip6tables-optimizer.
 
 .SH "AUTHOR"
-Johannes Hubertz <johannes@hubertz.de> wrote this in 2012 - 2015.
+Johannes Hubertz <johannes@hubertz.de> wrote this in 2012 - 2014.
 Anytime comments are welcome.
 
--- iptables-optimizer-0.9.13.orig/man/iptables-optimizer.8
+++ iptables-optimizer-0.9.13/man/iptables-optimizer.8
@@ -1,5 +1,5 @@
 
-.TH iptables-optimizer 8 "February 2015" "Johannes Hubertz" " "
+.TH iptables-optimizer 8 "September 2014" "Johannes Hubertz" " "
 
 .SH NAME
 iptables-optimizer \- optimize iptables filter-chains in kernel depending on their usage counters
@@ -114,6 +114,6 @@ GNU General Public License version 3 or
 iptables-optimizer.
 
 .SH "AUTHOR"
-Johannes Hubertz <johannes@hubertz.de> wrote this in 2012 - 2015.
+Johannes Hubertz <johannes@hubertz.de> wrote this in 2012 - 2014.
 Anytime comments are welcome.
 
